### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by elad**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Copyright (C) 2016-2017 Rob Wu rob@robwu.nl
```
Comment (Rob--W) [inline LICENSE]: Can you put angle brackets around the e-mail address?

And please edit the commit description to explicitly call this the MIT license.

Comment (elad) [inline LICENSE]: Oh, sure. I just copied it almost verbatim from another project of yours. :)

Comment (Rob--W) [inline LICENSE]: I guess you copied the rendered markdown? If you want the actual source via Github's interface, click on the "Raw" button/link and you will see the original source.

Comment (elad) [inline LICENSE]: I added the brackets. I'd rather not edit history just for mentioning the license type (but I did mention it in the second commit message). If you insist on mentioning "MIT license" in the first commit message let me know and I'll simply create a new PR.

**Comment by Rob--W** [comment]
Please squash the two commits into one (basically `git rebase -i` and change `pick` to `squash`, you can amend the commit description during rebase by using `reword` or with `git commit -v --am`; then force-push to Github using `git push -f` and the PR will automatically be updated). If you don't know how to do that, then I can do it.

**Comment by elad** [comment]
Please go ahead and do it. Thanks.

**Comment by Rob--W** [comment]
Thanks for the PR.

**Pull request merged**